proto/api_v2/sampling.proto

```diff
@@ -36,37 +36,53 @@ enum SamplingStrategyType {
   RATE_LIMITING = 1;
 };
 
+// ProbabilisticSamplingStrategy defines TraceIdRatioBased sampler configuration.
 message ProbabilisticSamplingStrategy {
+  // // Sampling rate between [0-1].
   double samplingRate = 1;
 }
 
+// RateLimitingSamplingStrategy defines RateLimiting sampling configuration.
 message RateLimitingSamplingStrategy {
+  // Traces per second.
   int32 maxTracesPerSecond = 1;
 }
 
+// OperationSamplingStrategy defines sampling configuration for a given operation/span name.
 message OperationSamplingStrategy {
+  // Operation name is a span name.
   string operation = 1;
+  // Probabilistic sampler configuration.
   ProbabilisticSamplingStrategy probabilisticSampling = 2;
 }
 
+// PerOperationSamplingStrategies defines sampling configuration for operations/span (names).
 message PerOperationSamplingStrategies {
+  // Default sampling probability for TraceIdRatioBased sampler.
```

Comment:
@yurishkuro I think comments in this type need more explanation. 

The per-operation config is clear - it is used if names match, if name does not match does it fallback to the probabilistic and rate limiting sampler? Is the rate limiting sampler used first until a specified throughput is satisfied and then it is switched to probabilistic?